src/Tasks/Backup/BackupJob.php

```diff
@@ -275,7 +275,11 @@ protected function copyToBackupDestinations(string $path)
     protected function sendNotification($notification)
     {
         if ($this->sendNotifications) {
-            event($notification);
+            try {
```

Comment:
Let's use the `rescue` helper instead of the try catch block: https://laravel.com/docs/master/helpers#method-rescue